views/leaderboard.pug

```diff
@@ -15,11 +15,37 @@ block content
   .container
     .row
       .col.s12
-        .card.center
-          .card-content
-            h4.center Leaderboard
-          .card-action
-            .leaderboard-content
+        .row
```

Comment:
what is leaderboard for?

Reply:
This gets filled with what ever fetchLeaderboard returns.